front/components/vaults/VaultDataSourceViewContentList.tsx

```diff
@@ -116,6 +116,37 @@ const getTableColumns = (showVaultUsage: boolean): ColumnDef<RowData>[] => {
   return columns;
 };
 
+function useStaticDataSourceViewHasContent({
```

Comment:
`nit`
We found team agreement to extract props in a dedicated interface 😉

Reply:
Hate it but will comply :)

Reply:
😆 at least the function's prototype is a bit cleaner and you don't need to scroll to read the actual code 🙃.

Reply:
Ok actually I just asked around and found 0 people in the office saying they agreed to this :p Are you sure there is an agreement here ?